src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -1635,11 +1647,20 @@ namespace RoomAir {
                     roomAFNNodeHVAC.SupplyFraction = ipsc->rNumericArgs(iEquipArg);
                     roomAFNNodeHVAC.ReturnFraction = ipsc->rNumericArgs(iEquipArg);
 
+                    // get equipment index
+                    int EquipIndex = 0;
+                    for (int thisZoneEquipNum = 1; thisZoneEquipNum <= state.dataZoneEquip->ZoneEquipList(iZone).NumOfEquipTypes;
+                         ++thisZoneEquipNum) {
+                        if (Util::SameString(state.dataZoneEquip->ZoneEquipList(iZone).EquipName(thisZoneEquipNum), roomAFNNodeHVAC.Name)) {
```

Comment:
This should also check `roomAFNNodeHVAC.zoneEquipType`. It's not common, but it is allowable for different types of zone equipment to have the same name.

Reply:
A good suggestion. Done!